applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -365,6 +365,7 @@ static void le_audio_evt_handler(enum le_audio_evt_type event)
 	int ret;
 
 	LOG_DBG("Received event = %d, current state = %d", event, strm_state);
+	uint32_t pres_delay;
```

Comment:
Would be great to move line 368 to line 366 and leave a blank after that.

Reply:
Hi @rick1082 - I've made no changes to that. I only wanted to add an option to enable active scanning. 

Is it easier for you if I make this change in this PR now?

Reply:
Done. I forgot that I made the changes to get it to compile. Yes, looks better that way.